api/json-vs-xml.md

```diff
@@ -0,0 +1,26 @@
+---
+description: >-
+  Choose between JSON and XML in the `api-connector` and `order-webhook-connector`
+---
+
+# JSON versus XML
+
+The Tradecloud API supports default a proprietary JSON format, but for some API endpoints a proprietary XML format is supported.
+
+## JSON
+
+The Tradecloud API supports default a proprietary Tradecloud [JSON format](requests.md#json-body), called `TSON`.
+
+All `api-connector`, `order-search`, `shipment-search`, `order-webhook-connector` and `shipment-webhook-connector` API endpoints support `TSON`.
```

Comment:
Do we really want to use `TSON`? imo it's funny, but not adding any value.
If anything, I'd say we adher to https://datatracker.ietf.org/doc/html/rfc7159, so it's just JSON.

For XML it's a bit different, as the notion of a "schema" is an intrinsic part of the syntax. In JSON, this is not the case afaik.

Reply:
I removed TSON, but just like our proprietary XML format, it is our proprietary JSON based format, its not just JSON.

Reply:
"proprietary JSON" sounds perfect to me.